src/content/docs/logs/log-management/log-api/log-event-data.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Log event data'
+tags:
+  - Logs
+  - Log management
+  - UI and data
+metaDescription: A summary of log data storage in New Relic.
+---
+
+Log events are records stored like any other event sent to New Relic. At a minimum the record must include:
+
+* `timestamp`
+* At least one attribute containing data
+
+Typically logs have a `message` field and level or severity, but we do not have fixed requirements for what you can send to New Relic.
+
+## Data storage [#events]
+
+Log records are stored by default in the `Log` event type. You can create additional event types by defining a custom data partition in Logs. The resulting types will always be prefaced with `Log_`. For detailed information, see our [data partitions documentation](/docs/logs/log-management/ui-data/data-partitions/).
+
+**Attributes:**
+
+* Maximum attributes: 255
+* Name: maximum 255 characters
+* Searchable storage: First 4096 characters
```

Comment:
Can we have it as "Searchable value"? And use bytes not chars

* Searchable value: The first 4096 bytes of data stored in an attribute can be queried directly in the UI